src/modules/common/components/Page.js

```diff
@@ -120,6 +133,7 @@ class Page extends React.Component<PropType, StateType> {
 
             onMessage={this.onMessage}
             style={{height: height}}
+            width={width}
```

Comment:
try to use the style attribute here for consistency

Reply:
Done.